examples/concurrent-mode/playroom.config.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+
+const braidDir = path.dirname(
+  require.resolve('braid-design-system/package.json')
+);
+
+module.exports = {
+  title: 'BRAID',
+  outputPath: './dist',
+  components: 'braid-design-system',
+  themes: 'braid-design-system/lib/themes',
+  frameComponent: './playroom/FrameComponent.js',
+  widths: [320, 500, 768, 1024],
+  snippets: './playroom/snippets.js',
+  unstable_concurrentMode: true,
+  exampleCode: `
+    <ChecklistCard>
+      <Checkbox id="1" label="This is a checkbox" checked={false} message={false} onChange={() => {}}>
+        <Text>
+          This text is visible when the checkbox is checked.
+        </Text>
+      </Checkbox>
+      <Checkbox id="2" label="This is a checkbox" checked message={false} onChange={() => {}}>
+        <Text>
+          This text is visible when the checkbox is checked.
+        </Text>
+      </Checkbox>
+      <Checkbox id="3" label="This is a checkbox" checked={false} message={false} onChange={() => {}}>
+        <Text>
+          This text is visible when the checkbox is checked.
+        </Text>
+      </Checkbox>
+    </ChecklistCard>
+  `,
+  webpackConfig: () => ({
+    resolve: {
+      alias: {
+        // These are done, because in example playroom _root_ imports them in its context, so it doesnt pick up the examples node_modules
+        react: require.resolve('react'),
+        'react-dom': require.resolve('react-dom'),
+      },
+    },
+    module: {
```

Comment:
this was all a copy-paste job of braid example. Let me know if you'd rather not. (braid is probably already concurrent mode compat, but if not i can use something else)